app-localize-behavior.html

```diff
@@ -227,6 +232,12 @@
     };
   },
 
+  __languageChanged: function(newLang, oldLang) {
+    if (newLang === oldLang) return;
```

Comment:
Is this `if` statement ever true? My understanding is observers don't fire unless there's an actual change to the value, so this will never actually happen